MaxText/decode.py

```diff
@@ -155,17 +158,17 @@ def prefill_or_load(config, model, state, rng, sp_tokenizer, mesh, state_mesh_sh
     blob = orbax_checkpointer.restore(config.prefill_cache_dir, restore_args=restore_args)
     blob["cache"] = jax.tree_map(lambda x : flax.linen.spmd.LogicallyPartitioned(x, mesh.axis_names), blob["cache"])
     max_logging.log(f"Restored prefill cache from {config.prefill_cache_dir}")
-    return blob["cache"], blob["next_token"], blob["pos"]
+    return blob["cache"], blob["last_logit"], blob["pos"]
   else:
-    cache, next_token, pos = compute_prefill(config, model, state, rng, sp_tokenizer, mesh,
+    cache, last_logit, pos = compute_prefill(config, model, state, rng, sp_tokenizer, mesh,
```

Comment:
Does the pos correspond to the last_logit here -> maybe call it last_pos for consistency.

Reply:
I believe it is next_pos. So I think last_logit and pos is ok? since they're off by 1?